src/components/nav/Nav.css

```diff
@@ -0,0 +1,48 @@
+@import url('https://fonts.googleapis.com/css2?family=Bodoni+Moda:ital,opsz,wght@0,6..96,400..900;1,6..96,400..900&family=Delicious+Handrawn&family=Enriqueta:wght@400;500;600;700&family=Lexend:wght@100..900&family=Macondo+Swash+Caps&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Raleway:ital,wght@0,100..900;1,100..900&display=swap');
+
+.nav-bar {
+    border-width: 4px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 5px;
+    min-height: fit-content; 
+    font-family: 'proxima-nova', sans-serif;
+}
+
+a {
+    text-decoration: none;
+    font-size: 1em;
+    color: #010101;
+    padding: 5px;
+}
+
+a:hover {
```

Comment:
LOVE hover styling! For our next iteration, how would yall feel about adding rules for `.active` so the nav bar reflects which page the user is on?

Reply:
Oh yes 100% plan on doing this!